### ISSUE TITLE
[bug] unvotable polls

### ISSUE THREAD

**Issue by mirabilos** [superseriousbusiness/gotosocial#2510]
### Describe the bug with a clear and concise description of what the bug is.

Polls from some instances cannot be voted on. They get a 422 status saying the poll is already closed. (The post JSON correctly says it’s not yet closed.)

Example, which is still open for another eight months so ample time for debugging if so inclined: https://0w0.is/objects/df60dd14-ddc3-4c5d-a7a8-dfb9c53f4b2d

Erm, well, `0w0.is` is going away soon, IIRC. So maybe not…

### What's your GoToSocial Version?

0.13.0-rc2 git-d0bb8f0

### GoToSocial Arch

amd64 binary

### What happened?

- Click on radiobutton corresponding to answer
- Click on 「Vote」 button
- get error

### What you expected to happen?

Be able to vote on polls that are not already closed.

**Comment by tsmethurst** [comment]
We've got a fix for this on main now :) https://github.com/superseriousbusiness/gotosocial/pull/2486

**Comment by mirabilos** [comment]
Oh, nice.

```
gotosocial=# UPDATE polls SET closed_at=NULL WHERE status_id='01HC9D5KF52YJXT5T970N9Z4EF';
UPDATE 1

tg@bove:~ $ sudo svc -t /service/GtS/                                                                           
```

… and it workarounds.

**Comment by tsmethurst** [comment]
Closing this as the fix is now in the 0.13.1 release :) Please feel free to open if you still see the bug; that may indicate another issue we're not aware of yet.
